### ISSUE TITLE
properties table XHTML output

### ISSUE THREAD

**Issue by AirQuick** [dita-ot/dita-ot#1890]
### Case 1
```
<properties>
	<property>
		<proptype>type</proptype>
		<propvalue>value</propvalue>
		<propdesc>desc</propdesc>
	</property>
</properties>
```
###### XHTML output:
```
<table border="1" cellpadding="4" cellspacing="0" class="simpletable properties" summary="">
	<tr>
		<th align="left" id="d27e13-type" valign="bottom">Type</th>
		<th align="left" id="d27e13-value" valign="bottom">Value</th>
		<th align="left" id="d27e13-desc" valign="bottom">Description</th>
	</tr>
	<tr class="strow property">
		...
```
###### Problem:
* `<thead>` and `<tbody>` do not exist.
* Header `<tr>` and `<th>` do not have @class.

### Case 2
```
<properties>
	<prophead>
		<proptypehd>typehd</proptypehd>
		<propvaluehd>valuehd</propvaluehd>
		<propdeschd>deschd</propdeschd>
	</prophead>
	<property>
		<proptype>type</proptype>
		<propvalue>value</propvalue>
		<propdesc>desc</propdesc>
	</property>
</properties>
```
###### XHTML output:
```
<table border="1" cellpadding="4" cellspacing="0" class="simpletable properties" summary="">
	<tr class="sthead prophead">
		<th align="left" class="stentry proptypehd" id="d27e17" valign="bottom">typehd</th>
		<th align="left" class="stentry propvaluehd" id="d27e20" valign="bottom">valuehd</th>
		<th align="left" class="stentry propdeschd" id="d27e23" valign="bottom">deschd</th>
	</tr>
	<tr class="strow property">
```
###### Problem:
* `<thead>` and `<tbody>` do not exist.

### Expected
In both cases I would expect `<thead>`, `<tbody>` and these @class'es:
```
<table border="1" cellpadding="4" cellspacing="0" class="simpletable properties" summary="">
	<thead>
		<tr class="sthead prophead">
			<th align="left" class="stentry proptypehd" id="d27e17-type" valign="bottom">Type</th>
			<th align="left" class="stentry propvaluehd" id="d27e17-value" valign="bottom">Value</th>
			<th align="left" class="stentry propdeschd" id="d27e17-desc" valign="bottom">Description</th>
		</tr>
	</thead>
	<tbody>
		<tr class="strow property">
```
### DITA-OT
2.1 M1

**Comment by eerohele** [comment]
eef15f8 adds the missing `<thead>` and `<tbody>` elements, only the missing `@class` attributes in Case 1 remain to be added.

**Comment by AirQuick** [comment]
If the missing `@class` were there, the user could simplify his CSS applying a base simpletable style.

**Comment by jelovirt** [comment]
Corrected in #911d5f1 and earlier commits.
